### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by erupomare**
Set up new files and folders for project structure. Included setup.py file.

### CODE REVIEW

**Comment by chrispcharlton** [comment]
Nice work! Smashed the virtual champagne bottle against the side of this project

**Review by chrispcharlton — APPROVED** [review]

**Comment by chrispcharlton** [comment]
@herector8 [see here for purpose of setup.py](https://packaging.python.org/tutorials/packaging-projects/)

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import setuptools
```
Comment (herector8) [inline setup.py:1]: What is the purpose of setuptools?

**Review by herector8 — APPROVED** [review]
Nice work!

File: setup.py (hunk 1 above)
Comment (chrispcharlton) [reply setup.py:1]: setuptools is the built-in python utility for packaging projects

**Review by herector8 — APPROVED** [review]

**Pull request merged**